samples/openvx/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 2.8.9)
+
```

Comment:
If there's no OpenVX then the directory `samples/openvx` won't be added as it's written in `samples/CMakeLists.txt`.